src/style.css

```diff
@@ -1,3 +1,71 @@
-body {
+* {
   margin: 0;
+  padding: 0;
+}
```

Comment:
![image](https://github.com/mate-academy/layout_moyo-header/assets/80700247/4c7334c2-8601-4539-9ce8-1dbd535aa601)

Use * selector only for box-sizing: border-box rule